lib/metasploit/framework/login_scanner/smb.rb

```diff
@@ -120,7 +120,7 @@ def attempt_login(credential)
             case status_code.name
               when *StatusCodes::CORRECT_CREDENTIAL_STATUS_CODES
                 status = Metasploit::Model::Login::Status::DENIED_ACCESS
-              when 'STATUS_SUCCESS'
+              when 'STATUS_SUCCESS', 'STATUS_PASSWORD_MUST_CHANGE'
```

Comment:
I think since `STATUS_PASSWORD_MUST_CHANGE` is still in `CORRECT_CREDENTIAL_STATUS_CODES` this will still report as `Metasploit::Model::Login::Status::DENIED_ACCESS` since that test is earlier in the `when` clauses of the `case` statement.

Reply:
good point